templates/githubpullrequest.mustache

```diff
@@ -17,7 +17,7 @@
             <a href="{{html_url}}" target="_blank" rel="noopener">{{title}}</a>
           </h4>
         </summary>
-        <p class="onebox-details-body">{{body}}</p>
+        <code class="onebox-details-body">{{{body}}}</code>
```

Comment:
What happens if someone were to put `</code><script>alert("bad things")</script>` in the body... 🤔

Reply:
I think maybe we should keep the double curleys, and instead use CSS to set `white-space: pre-wrap`?

Reply:
I don't think it would work, but let me try.